tests/basic.rs

```diff
@@ -125,3 +125,55 @@ fn command_macro() {
     assert_eq!(state.x, 29);
     assert_eq!(state.player.as_str(), "twenty-sixPLAYER");
 }
+
+#[test]
+fn help_command() {
+    use std::borrow::Cow;
+    use std::rc::Rc;
+    struct State {
+        pub dispatcher: Rc<CommandDispatcher<Self>>,
+        pub usages: Vec<Cow<'static, str>>,
+        pub descriptions: Vec<Cow<'static, str>>,
+    }
+
+    let mut dispatcher = CommandDispatcher::new();
+
+    #[command(
+        usage = "help <page>",
+        description = "Shows the descriptions and usages of all commands."
+    )]
+    fn help(state: &mut State, page: u32) {
+        state.usages = state
```

Comment:
I think this will panic if `page` is out of bounds; we should probably do a check.

Reply:
Help operates on the iterator of `CommandSpec` and as such just returns an empty array instead of panicking.